pkg/alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # image was bootstraped using FROM lfedge/eve-alpine-base:fad44e3702708a8d044663a20fd98d933dddb41e AS cache
 # to update please see https://github.com/lf-edge/eve/blob/master/docs/BUILD.md#how-to-update-eve-alpine-package
-FROM lfedge/eve-alpine:82df60e43ab9f8c935584b8c7b4d0a4b0271d608 AS cache
+FROM lfedge/eve-alpine:2ed33236f1eea746532f612d6c4b7779c5da2193 AS cache
```

Comment:
But why should we use an eve-alpine version different from the standard one (82df60e43ab9f8c935584b8c7b4d0a4b0271d608)?

Reply:
`82df60e43ab9f8c935584b8c7b4d0a4b0271d608` isn't a "standard" one. It is a bit long to put in a comment here, but it is better laid out in the [ALPINE.md](https://github.com/lf-edge/eve/blob/master/docs/ALPINE.md) document.

In short, each alpine image is based on the previous one. So every time you change alpine packages or Dockerfile, you should change `FROM <alpine>` to whatever the previous one was. Sort of:

1. `current=$(lkt pkg show-tag pkg/alpine)`
2. `git checkout -b new-branch`
3. change `pkg/alpine/Dockerfile` line `FROM` to be `${current}`
4. make other changes
5. commit, push, open PR, etc.

Does that help?

There is a lot more detail in [ALPINE.md](https://github.com/lf-edge/eve/blob/master/docs/ALPINE.md). I really want to make this caching solution easier to understand and simpler, but haven't come up with a good alternative yet.

Reply:
Ah, sorry, I'm stupid. We change exactly the alpine image here and just update the hash after that...
So after this PR is merged, we should also update the hashed in the other packages that use it, right?...

Reply:
> So after this PR is merged, we should also update the hashed in the other packages that use it, right?...

Thanks to `get-deps` it can already be don in this PR.

Reply:
Yep, it would be nice to bump the version as the second commit in this PR.

Reply:
> Ah, sorry, I'm stupid. 

No, not at all. Just this thing is confusing.

> We change exactly the alpine image here and just update the hash after that...
> So after this PR is merged, we should also update the hashed in the other packages that use it, right?..

Up to us. In theory, yes, we should, but we don't _have_ to until we need it.

Reply:
> get-deps it can already be don in this PR

I don't understand. `get-deps` is a binary tool, built and invoked by Makefile. What has to change with get-deps based on this PR?

Reply:
> > Ah, sorry, I'm stupid.
> 
> No, not at all. Just this thing is confusing.
> 
> > We change exactly the alpine image here and just update the hash after that...
> > So after this PR is merged, we should also update the hashed in the other packages that use it, right?..
> 
> Up to us. In theory, yes, we should, but we don't _have_ to until we need it.

It will save us time figuring out the reason for inconsistencies later, so I would prefer to do it as soon as the Alpine package is updated.

Reply:
@deitch `get-deps` creates the dependencies for the Dockerfiles, so that you don't have to push first to docker hub, but you can use the local image already (because `get-deps` organized to build that one before)

Reply:
> Or do you mean it's possible to update deps like installer in the same PR and it will build it for consumption locally?

Yes.